src/scrapeCalendar.ts

```diff
@@ -232,11 +236,31 @@ function fillEntry(
   };
 }
 
-function recognizeCalendarDate(date: CalendarDate): RecognizerRules {
+function recognizeCalendarDate(
+  date: CalendarDate,
+  fieldRainoutInfo: { [key: string]: boolean },
+): RecognizerRules {
   const intervals: RuleInterval[] = [];
   let lastEntry: ParsedName | null = null;
+  const isFieldRainedOut = fieldRainoutInfo[date.date] || false;
+
+  if (isFieldRainedOut) {
+    // if the field is rained out, override the entries
+    const dateWithTime = `${date.date}T05:00:00`;
+    const parsedDate = new Date(dateWithTime);
+    date.entries = [
+      {
+        name: OPEN_ALL_DAY,
+        startDate: dateWithTime,
+        description: "",
+        subHeaderDate: formatDate(parsedDate, "MMMM d, yyyy K:mm aa"),
```

Comment:
My primary concern with this is that it will render incorrectly when run on the edge (in UTC) since date-fns is completely ignorant of time zones and date-fns-tz doesn't support date-fns 3 yet.